src/screens/GroupSetting/elements/EditGroup.js

```diff
@@ -48,7 +48,7 @@ const EditGroup = ({
     }
     return (
       <View testID="imagePencil">
-        <MemoIc_pencil width={25} height={25} color={COLORS.lightgrey} />
+        <MemoIc_pencil width={25} height={25} color={COLORS.gray100} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIc_pencil` component has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could impact visibility and user experience.
```diff
- <MemoIc_pencil width={25} height={25} color={COLORS.lightgrey} />
+ <MemoIc_pencil width={25} height={25} color={COLORS.gray100} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->